pkg/virt-launcher/virtwrap/api/BUILD.bazel

```diff
@@ -31,7 +31,13 @@ go_test(
         "defaults_test.go",
         "schema_test.go",
     ],
+    data = glob(["testdata/**"]),
```

Comment:
Same here: probably not needed.

Reply:
I'm seeing that `testdata`  is not random name, but actually treated specially. Seems Go [ignores](https://pkg.go.dev/cmd/go#hdr-Test_packages) testdata directories.

> The go tool will ignore a directory named "testdata", making it available to hold ancillary data needed by the tests. 

That's likely why `testdata` generates different bazel build then when I had `embed` directory.